### ISSUE TITLE
Re-use schema.org schemas

### ISSUE THREAD

**Issue by blootsvoets** [eStep/eStep#1]
Briefly looking at JSON-LD and @context and @type, for our naming conventions in our schema's, would it be useful to look at http://schema.org?

E.g.
Software: https://schema.org/SoftwareApplication
Person: https://schema.org/Person
Organization: https://schema.org/Organization

The Project schema does not exist, nor does a Collaboration or Funding (I couldn’t think of a good synonym), that could be a good contribution?

**Comment by mkuzak** [comment]
+ 1 one for using schema.org

**Comment by blootsvoets** [comment]
Btw, stackoverflow and this very page also use schema.org:
```
<div itemscope="" itemtype="http://schema.org/SoftwareSourceCode">
```

**Comment by mkuzak** [comment]
schema.org is very actively developed project, there is even project for life sciences 'BioSchemas' for bio-related metadata. Search engines use it to to reason about the pages,
all the summaries that we see in google results on the right side are generated based on schema.org used by indexed websites.
There is very handy tool from google to test schema use:
https://developers.google.com/structured-data/testing-tool/

**Comment by wrvhage** [comment]
Yes. As soon as we start with the JSON-LD additions, we should add the appropriate Schema.org annotations.
